ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -271,15 +276,22 @@ Deploy Model Via CLI
 Get KServe Inference Host Via CLI
     [Arguments]    ${isvc_name}    ${namespace}
     ${rc}    ${ksvc_host}=    Run And Return Rc And Output
-    ...    oc get ksvc ${isvc_name}-predictor -n ${namespace} -o jsonpath='{.status.url}' | cut -d'/' -f3
+    ...    oc get isvc ${isvc_name} -n ${namespace} -o jsonpath='{.status.url}' | cut -d'/' -f3
+    # ...    oc get ksvc ${isvc_name}-predictor -n ${namespace} -o jsonpath='{.status.url}' | cut -d'/' -f3
     Should Be Equal As Integers    ${rc}    ${0}
     Should Not Be Empty    ${ksvc_host}
     RETURN    ${ksvc_host}
 
+Get Current Revision ID
```

Comment:
I think it will be better to rename the keyword to "Get POD Current Revision ID"